build-support/githooks/pre-commit

```diff
@@ -2,6 +2,10 @@
 
 set -e
 
+# If you use the `pants` scie, you may "export PANTS=pants" in order to use the same bootstrapped
+# pants setup for this pre-commit hook as you get from the command line for any other pants command.
+: "${PANTS:=./pants}"
```

Comment:
This is a fairly generic environment variable name... I haven't seen it used before, but it might be good to use something more specific and greppable.

Reply:
yea, and it's really local to the pants repo only too, so `_PANTS_PRE_COMMIT_BINARY` seems fair. Thanks.